Assets/Scripts/Boids/SpawnRandomInSphereSystem.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+using Boids;
+using Unity.Collections;
+using Unity.Entities;
+using Unity.Mathematics;
+using Unity.Transforms;
+
+namespace Common
+{
+    [UpdateInGroup(typeof(SimulationSystemGroup))]
+    [UpdateAfter(typeof(TransformSystemGroup))]
+    public class SpawnRandomInSphereSystem : ComponentSystem
+    {        
+        protected override void OnUpdate()
+        {
+            Entities.ForEach((Entity e, SpawnRandomInSphere spawner, ref LocalToWorld localToWorld) =>                
+            {
```

Comment:
Is this running in Parallel. If so, we might be able to speed it up more by moving it to the GPU.